src/pages/instances/forms/EditInstanceDetails.tsx

```diff
@@ -33,7 +33,7 @@ interface Props {
 }
 
 const EditInstanceDetails: FC<Props> = ({ formik, project }) => {
-  const isReadOnly = formik.values.readOnly;
+  const readOnly = formik.values.readOnly;
```

Comment:
how did this work before? InstanceEditDetailsFormValues interface only had readOnly, is it considering property from another unioned interface?

Reply:
Yes, the readOnly is in the `InstanceEditDetailsFormValues` type.